hera_pspec/grouping.py

```diff
@@ -153,6 +154,14 @@ def average_spectra(uvp_in, blpair_groups=None, time_avg=False,
         does this for every specified key. Every stats_array key that is
         not specified is thrown out of the new averaged object.
 
+    error_weights: string, optional
+         error_weights specify which kind of errors we use for weights 
+         during averaging power spectra.
+         The weights are defined as $w_i = 1/ sigma_i^2$, 
+         where $sigma_i$ is taken from the relevant field of stats_array.
+         If `error_weight' is set to None, which means we 
+         just use the integration time as weights. Default: None
```

Comment:
should add that if `error_weights` is specified then it also gets assigned as `error_field`, and if one specified both then `error_weights` supercedes as `error_field`. lmk if that makes sense.

Reply:
My understanding is that the elements in error_field include error_weights, which allow us to also weight and average other error bars with the specified one.